.github/workflows/release.yml

```diff
@@ -96,6 +96,18 @@ jobs:
           echo '::set-env name=target-directory::.build/woocommercev4'
           echo '::set-env name=build-command::make woocommercev4-build'
 
+      - name: Prepare release for WoocommerceV5
+        if: startsWith(github.ref, 'refs/tags/WoocommerceV5-')
+        run: |
+          PACKAGE_VERSION=${{ steps.packageVersion.outputs.tag }}
+          echo 'extensions=curl, json, mbstring, xdebug' >> $GITHUB_ENV
+          echo "release-name=woocommerce_v4-${PACKAGE_VERSION//\./_}" >> $GITHUB_ENV
+          echo 'php-versions=7.2' >> $GITHUB_ENV
+          echo 'key=cache-woocommercev5-v2' >> $GITHUB_ENV
```

Comment:
`cache-woocommercev5-v1'

The reason that i put `v2` on some package releases is due to legacy reason (I did cache using `v1` previously but some bugs). New package should use `v1`